adapter/repository/in-memory-simple/src/test/java/org/mobilitydata/gtfsvalidator/db/NoticeMemoryConsumptionTest.java

```diff
@@ -0,0 +1,151 @@
+package org.mobilitydata.gtfsvalidator.db;
+
+import org.junit.jupiter.api.Test;
+import org.mobilitydata.gtfsvalidator.usecase.notice.IntegerFieldValueOutOfRangeNotice;
+
+public class NoticeMemoryConsumptionTest {
+
+    private void generateNotices(InMemoryValidationResultRepository resultRepository, int numberOfNotices) {
+        for (int i = 0; i < numberOfNotices; i++) {
+            resultRepository.addNotice(new IntegerFieldValueOutOfRangeNotice("filename",
+                    "fieldname",
+                    "entity_id",
+                    0,
+                    100,
+                    101)
+            );
+        }
+    }
+
+    @Test
+    public void creationOf100NoticeShouldNotExceedMemoryLimit() {
+
+        long totalMemory = Runtime.getRuntime().totalMemory(); // bytes
+
+        InMemoryValidationResultRepository resultRepository = new InMemoryValidationResultRepository();
+
+        generateNotices(resultRepository, 1);
+        resultRepository.getAll();
+
+        long freeMemory = Runtime.getRuntime().freeMemory();  // bytes
+
+        System.out.println(
+                String.format("Generating 100 notices: Total memory: %s megabytes, Free memory: %s megabytes, Used memory: %s megabytes",
```

Comment:
It doesn't seem like this is showing up in the Travis builds (https://github.com/MobilityData/gtfs-validator/runs/519805000). Are we using a logging framework that should be used here instead?

Reply:
We are using LogManager in the main method. Will use this framework here to enable the display of this information in the CI builds